GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -276,7 +288,9 @@ public int compare(String o1, String o2) {
                         + " - instance: " + serverBaseUrl);
             }
         } catch (Exception e) {
+
             log.error("Error generating report: " + e.getMessage(), e);
+            e.printStackTrace();
```

Comment:
Can also be added to debug logging maybe?

Reply:
Like this?

Reply:
Do we need the entire stacktrace or is the message enough? if so then we go with just the message.

Reply:
If it breaks it would be good to know exactly where.

Reply:
If all we get is an NPE somewhere in all the xml classes, that is hard to work with.

Reply:
Most of Flow lets exceptions propagate all the way out so you get a stackdump that way.